roles/matterbridge/vars/main.yml

```
@@ -1,67 +1,140 @@
 ---
-default_irc_bot_nick: slack-ritlug
+
+# these values are used mainly in defining the networks in the config (i.e. what sources of channels exist)
+
+default_irc_bot_nick: foss-rit-bridge
```

```suggestion
default_irc_bot_nick: rit-foss-bridge
```

A minor nitpick because of branding things. Trying to lean into using "RIT FOSS" or "RIT Open Source Community" where possible.